src/components/common/ErrorBoundry.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { Component, ErrorInfo, ReactNode } from 'react'
+import * as Sentry from '@sentry/browser'
+
+interface Props {
+  children?: ReactNode
+}
+
+interface State {
+  hasError: boolean
+}
+
+class ErrorBoundary extends Component<Props, State> {
+  public state: State = {
+    hasError: false,
+  }
+
+  public static getDerivedStateFromError(): State {
+    // Update state so the next render will show the fallback UI.
+    return { hasError: true }
+  }
+
+  public componentDidCatch(error: Error, errorInfo: ErrorInfo) {
+    Sentry.captureException(error, {
+      tags: {
+        client_side_exception: errorInfo.componentStack,
+      },
+    })
+
+    console.error('Uncaught error:', error, errorInfo)
+  }
+
+  public render() {
+    if (this.state.hasError) {
+      return <h1>Sorry.. there was an error</h1>
```

Comment:
Trans tags aren't instantiated yet. Assuming just having English copy is alright for the moment. 


The other thought I had was keeping this Error boundary as a last case error catch and adding another error boundary that's inside the i18n provider.